app/src/main/java/com/tnco/runar/ui/activity/MainActivity.kt

```diff
@@ -38,7 +41,30 @@ class MainActivity : AppCompatActivity(), Navigator, AudioManager.OnAudioFocusCh
 
     private lateinit var firebaseAnalytics: FirebaseAnalytics
 
+    private val fragmentListener =
```

Comment:
added a listener to detect the moment when the fragment is fully rendered, so that the navigation bar can be displayed